be/src/storage/update_manager.cpp

```diff
@@ -283,6 +283,46 @@ void UpdateManager::clear_cached_delta_column_group(const std::vector<TabletSegm
     }
 }
 
+Status UpdateManager::set_cached_empty_delta_column_group(KVStore* meta, const TabletSegmentId& tsid) {
```

Comment:
Why to need to cache a empty delta group?

Reply:
Each time we create a segment iterator, we need to find out that this segment has a delta column group or not. So if we don't cache a empty delta group, that we have to read from rocksdb which is more slow.